tests/bluetooth/tester/default.conf

```diff
@@ -9,5 +9,17 @@ CONFIG_BT_ATT_PREPARE_COUNT=4
 CONFIG_BT_GATT_CLIENT=y
 CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
 CONFIG_BT_PRIVACY=y
+
+CONFIG_BT_MESH=y
+CONFIG_BT_MESH_PROV=y
+CONFIG_BT_MESH_RELAY=y
+CONFIG_BT_MESH_PB_ADV=y
+CONFIG_BT_MESH_PB_GATT=y
+CONFIG_BT_MESH_GATT_PROXY=y
+CONFIG_BT_MESH_SUBNET_COUNT=5
+CONFIG_BT_MESH_APP_KEY_COUNT=5
```

Comment:
Same with app keys, I think 1 should be sufficient (unless you can point to a test case that needs more)

Reply:
ok, I'll use default values then. We can always change it if there will be need